### DISCUSSION TITLE
move the popup label (animated) together with marker.

### DISCUSSION THREAD

**Discussion by IwanBrun** [Leaflet/Leaflet#7890] [Q&A] [answered]
Is it possible to animated-move the popup (label) with the animated marker?
on L.animatedMarker.setLatLng(newLatLng); the popup jumps to the destination first and then de marker smoothly moves to the new LatLon position.
it would be so nice if they move together to the new endpoint, is there an easy solution for this?
Thanks!

**Comment by Falke-Design** [comment, marked as answer]
This is the Leaflet core repository, we can't help you with plugin specific issues. Please ask this question the on the plugin repository [Leaflet.AnimatedMarker](https://github.com/openplans/Leaflet.AnimatedMarker) or on [StackOverflow](https://stackoverflow.com/questions/tagged/leaflet).
